common/session.go

```diff
@@ -98,15 +98,19 @@ func (s *Session) readLoop() {
 		select {
 		case msg := <-s.readCh:
 			ev, err := cdproto.UnmarshalMessage(msg)
+			if errors.Is(err, cdp.ErrUnknownCommandOrEvent("")) && msg.Method == "" {
```

Comment:
I don't think this `errors.Is()` check will work. `Is()` checks against a specific value (e.g. [`ErrNotFound`](https://pkg.go.dev/os/exec#pkg-variables)), whereas `As()` checks against a specific type, which [`ErrUnknownCommandOrEvent`](https://github.com/chromedp/cdproto/blob/master/cdp/types.go#L71) is. So we should use `errors.As()` here.

Reply:
It works because that's a conversion, not a new error.

Reply:
Ah, so you're checking that we received an empty `ErrUnknownCommandOrEvent`. OK, it makes sense after reading the comment below.